tests/game/test_buildings.py

```diff
@@ -20,6 +20,7 @@
 # ###################################################
 
 from itertools import product
+import pytest
```

Comment:
That's interesting, I would have thought pytest enforces a new line between groups. Weird.